app/views/feedback/steps/_website_experience.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= f.govuk_text_area :unsuccessful_visit_explanation, label: { tag: 'h1', text: "What would you like to tell us about your experience on our website? "} do %>
+<p>
+  You can give us feedback on what we’re doing well, as well as ways we could improve.
```

Comment:
```suggestion
  You can give us feedback on what we’re doing well and how we could improve.
```

Reply:
This will become irrelevant if we simplify the questions